### PR TITLE
Add Linting

### PR DESCRIPTION
**Pull request by BenjaminAleritsch**
Add linting via .github/workflows/main.yml

### CODE REVIEW

File: .yamllint [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+extends: default
+ignore: |
+  .github
+  meta*
+  roles/elasticsearch/
```
Comment (Fobhep) [inline .yamllint]: that line is probably copy/pase left over ;)

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+ansible>=2.7.5
+yamllint
+flake8
```
Comment (Fobhep) [inline requirements.txt]: I don't think you need flake8 here

**Review by Fobhep — CHANGES_REQUESTED** [review]

File: .yamllint (hunk 1 above)
Comment (BenjaminAleritsch) [reply .yamllint]: fixed

File: requirements.txt (hunk 2 above)
Comment (BenjaminAleritsch) [reply requirements.txt]: fixed

**Review by Fobhep — APPROVED** [review]

**Pull request merged**